### PR TITLE
Refactor test after split lanes issue

### PR DESCRIPTION
**Pull request by marioferh**
Tests: Updating of nodeSelector parameter and node labels

Is to complicated, takes a lot of time (around 40min) and fails because timeout of the suite many times.

Fastest solution its to move it to functest 7_ after the split https://github.com/openshift/cluster-node-tuning-operator/pull/536

Before Fix:
ci/prow/e2e-gcp-pao Takes around 1:40 min
ci/prow/e2e-gcp-pao-updating-profile Takes > 4h = timeout Fails

After Fix:
ci/prow/e2e-gcp-pao Takes around 2:20 min
ci/prow/e2e-gcp-pao-updating-profile Takes 3:40 < 4h timeout Works

Signed-off-by: Mario Fernandez <mariofer@redhat.com>

### CODE REVIEW

File: test/e2e/performanceprofile/functests/2_performance_update_second/updating_profile_mcp_labels.go [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+		performanceMCP, err = mcps.GetByProfile(profile)
+		Expect(err).ToNot(HaveOccurred())
+
+		// Verify that worker and performance MCP have updated state equals to true
+		for _, mcpName := range []string{testutils.RoleWorker, performanceMCP} {
+			mcps.WaitForCondition(mcpName, machineconfigv1.MachineConfigPoolUpdated, corev1.ConditionTrue)
+		}
+	})
+
+	Context("Updating of nodeSelector parameter and node labels", func() {
```
Comment (marioferh) [inline test/e2e/performanceprofile/functests/2_performance_update_second/updating_profile_mcp_labels.go:58]: This context takes around 40-50 min. Even if with this "patch" it works a refactor is required.

**Comment by yanirq** [comment]
/close
refactor will happen in different PRs

**Comment by jmencak** [comment]
/cc @eggfoobar 
The work that was happenning to address the `e2e-gcp-pao-updating-profile` issues.

**Pull request closed without merging**